tests/test_action_inputs.py

```diff
@@ -157,3 +160,37 @@ def test_get_duplicity_scope_wrong_value(mocker):
 
     assert ActionInputs.get_duplicity_scope() == "BOTH"
     mock_error.assert_called_with("Error: '%s' is not a valid DuplicityType.", "HUH")
+
+
+def test_detect_row_format_invalid_keywords_no_invalid_keywords(caplog):
+    caplog.set_level(logging.ERROR)
+    row_format = "{number} _{title}_ in {pull-requests}"
+    ActionInputs._detect_row_format_invalid_keywords(row_format)
+    assert len(caplog.records) == 0
+
+
+def test_detect_row_format_invalid_keywords_with_invalid_keywords(caplog):
+    caplog.set_level(logging.ERROR)
+    row_format = "{number} _{title}_ in {pull-requests} {invalid_key} {another_invalid}"
+    ActionInputs._detect_row_format_invalid_keywords(row_format)
+    assert len(caplog.records) == 2
+    expected_errors = [
+        "Invalid `invalid_key` detected in `Issue` row format keyword(s) found: invalid_key, another_invalid. Will be removed from string.",
+        "Invalid `another_invalid` detected in `Issue` row format keyword(s) found: invalid_key, another_invalid. Will be removed from string."
+    ]
+    actual_errors = [record.getMessage() for record in caplog.records]
+    assert actual_errors == expected_errors
+
+
+def test_clean_row_format_invalid_keywords_no_keywords():
+    row_format = "{number} _{title}_ in {pull-requests}"
+    cleaned_format = ActionInputs._clean_row_format_invalid_keywords(row_format)
+    assert cleaned_format == row_format
```

Comment:
Add before these actual and expected please.

Reply:
Test improved in commit 3a77a9ec12c250ead4ad816666a634511d384a2e.